info.json

```diff
@@ -1,20 +1,19 @@
 {
 	"name": "laserfence",
-	"title": "⚡ C&C Laserfence",
-	"description": "⚡Use Laser Fence technology that was developed during the Tiberium Wars to fend off biters.",
+	"title": "Laser Fence",
```

Comment:
the unicode emojie look so cool in mod menu ;_; you don't like it ?

Reply:
I like the name as simply "Laser Fence"
I added Command and Conquer to the description to make it more searchable and you can title the French version however you want